### DISCUSSION TITLE
pv database_name.gz | gunzip -c | warden db import Command not working

### DISCUSSION THREAD

**Discussion by amit-ranosys** [wardenenv/warden#809] [Help & Support]
Hi,

warden db import command not working giving the below error

ERROR at line 1: Unknown command '\-'.                                                                                                 ]   0% ETA 0:29:11
ERROR: Command `"$WARDEN_BIN" env exec -T db mysql -u"${MYSQL_USER}" -p"${MYSQL_PASSWORD}" --database="${MYSQL_DATABASE}" "${WARDEN_PARAMS[@]:1}" "$@"` at /opt/warden/commands/db.cmd:39 failed with exit code 1
 576KiB 0:00:01 [ 522KiB/s] [>

**Comment by bap14** [comment]
@amit-ranosys  What does the first line of your DB dump look like? What is the full command you used to invoke the import process?

**Reply by amit-ranosys** [reply]
@bap14 I am using below command to import db
`pv database_name.gz | gunzip -c | warden db import`
